internal/pkg/acl/acl.go

```diff
@@ -205,7 +212,7 @@ func convertToFlags(operations ...any) string {
 	return strings.Join(ops, ", ")
 }
 
-func ConvertToLower(operations ...any) string {
+func ConvertToLower[T any](operations []T) string {
```

Comment:
Wouldn't this work?
```suggestion
func ConvertToLower(operations []any) string {
```

Reply:
To do that I have to also change `AclOperations = []mds.AclOperation{...` to `AclOperations = []any{...` to get it compiling; this version just seems cleaner to me.